src/IECoreRI/RendererImplementation.cpp

```diff
@@ -278,6 +278,18 @@ void IECoreRI::RendererImplementation::setOption( const std::string &name, IECor
 		return;
 	}
 
+
+	if( name == "render:sampleMotion" )
```

Comment:
This shouldn't have the "render:" prefix, as I explained in previous emails. Options destined to be supported in all renderers (like this one) are unprefixed. Renderer specific options are prefixed.